lib/ex_doc/translations.ex

```diff
@@ -0,0 +1,401 @@
+defmodule ExDoc.Translations do
```

Comment:
As a general rule module names should be in singular.
```suggestion
defmodule ExDoc.Translation do
```